src/vl-mapactions-snap-interaction.js

```diff
@@ -1,12 +1,9 @@
 import Snap from 'ol/interaction/Snap';
 
 export class VlSnapInteraction extends Snap {
-  constructor(layer) {
-    const source = layer ? layer.getSource() : null;
-    const snapOptions = {
-      source: source,
-      pixelTolerance: 7,
-    };
+  constructor(options) {
+    const snapOptions = Object.assign({}, options);
+    snapOptions.pixelTolerance = options.pixelTolerance || 7;
     super(snapOptions);
     this.snapOptions = snapOptions;
   }
```

Comment:
Breaking change van layer naar options argument, best vermelden in issue zodat hiermee rekening gehouden kan worden bij release.

Reply:
Eventueel ook overwegen op layer argument te behouden en een optioneel 2e options argument introduceren, dan zijn er geen breaking changes. Als dat niet mogelijk is, zijn de breaking changes op zich ook geen probleem.

Reply:
Alles is mogelijk maar leek we gewoon wat cleaner ...